### PR TITLE
Fix for-in eslint errors in client-debugger

### PR DESCRIPTION
**Pull request by knightburton**
IoT.jsCode-DCO-1.0-Signed-off-by: Imre Kiss kissi.szeged@partner.samsung.com

### CODE REVIEW

File: src/app/client-debugger.js [hunk 1]
```diff
@@ -328,44 +328,46 @@ export default class DebuggerClient {
     }
 
     for (let i in format) {
-      if (format[i] === 'B') {
-        result.push(message[offset]);
-        offset++;
-        continue;
-      }
+      if (format.hasOwnProperty(i)) {
+        if (format[i] === 'B') {
```
Comment (rtakacs) [inline src/app/client-debugger.js]: I'd use the following code:

```js
if (format.hasOwnProperty(i) == false)
  continue;
```

File: src/app/client-debugger.js [hunk 2]
```diff
@@ -1031,19 +1033,21 @@ export default class DebuggerClient {
     let length = 0;
 
     for (const i in format) {
-      if (format[i] === 'B') {
-        length++;
-        continue;
-      }
+      if (format.hasOwnProperty(i)) {
+        if (format[i] === 'B') {
```
Comment (rtakacs) [inline src/app/client-debugger.js]: Ditto.

File: src/app/client-debugger.js (hunk 1 above)
Comment (knightburton) [reply src/app/client-debugger.js]: In this case I think that we should not change this, based on the ESLint rules.

File: src/app/client-debugger.js (hunk 2 above)
Comment (knightburton) [reply src/app/client-debugger.js]: Same as above.

**Comment by rtakacs** [comment]
LGTM

**Comment by yichoi** [comment]
LGTM

**Pull request merged**